tests/garage/envs/test_gray_scale_env.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+import gym
+import numpy as np
+
+from garage.envs.wrappers import GrayScale
+from garage.misc.overrides import overrides
+
+
+class TestGrayScale(unittest.TestCase):
+    @overrides
+    def setUp(self):
+        self.env = gym.make("Breakout-v0")
+        self.env_r = GrayScale(gym.make("Breakout-v0"))
```

Comment:
By including gym.make you accidentally made this an integration test.

use a mock to provide a known-value input instead.

https://semaphoreci.com/community/tutorials/getting-started-with-mocking-in-python